testing_and_setup/compass/load_compass_env.bash

```diff
@@ -0,0 +1,36 @@
+version=0.1.2
+
+default_python=3.7
```

Comment:
This doesn't look like it's used in the script right now. This must be needed as an environment variable?

Reply:
Good call, that's an artifact from the script I adapted this from.